terraform/modules/single_instance/variable.tf

```
@@ -58,6 +62,10 @@ variable "rootdisk_size" {
 variable "is_windows" {
   type = bool
 }
+
+variable "persistent_root" {
+  type = bool
```

This should have a default
```suggestion
  type = bool
  default = false
```